wiki/Guides/Audio_Editing/en.md

```diff
@@ -8,27 +8,32 @@ This article serves as guide to help you do minor edits to your audio files for
 
 ## Audacity (and LAME)
 
-[Audacity](https://www.audacityteam.org/download) is a open source free audio editing and recording software. To properly use this for `.mp3`, you will need to use LAME.
+[Audacity](https://www.audacityteam.org/download) is a free, open-source audio editing and recording software. It uses the [LAME](https://lame.sourceforge.io) encoding library to be able to export audio in the MP3 audio format.
 
-[LAME](https://lame.sourceforge.io) is an `.mp3` encoding library that will allow Audacity to export sound files in the `.mp3` format whilst using different bit rates. To install LAME on Audacity, refer to the [Audacity wiki](https://manual.audacityteam.org/man/faq_installing_the_lame_mp3_encoder.html).
+Since 2017, after the last few patents on the MP3 audio format had expired, the LAME encoding library started to be included in later versions of Audacity on Windows and macOS, thus manual installation is no longer necessary.
```

Comment:
this bit of history is a bit off topic for the wiki

Reply:
I think you're right, maybe just make it shorter to at least say that LAME is built into Audacity on Windows and macOS, so the next paragraph for Linux users would make sense.

Reply:
Or maybe just delete it outright, I'm not sure which is best so maybe up to you?